### DISCUSSION TITLE
Broadcast sound is missing in a few hours after server startup

### DISCUSSION THREAD

**Discussion by bigsssss** [OvenMediaLabs/OvenMediaEngine#1232] [Help]
Video stream is transmitted by SRT, video transcoding is disabled, only OPUS audio transcoding is used. Video viewing only via WebRTC. After some time after starting the server, the sound on the video stream disappears completely, while the video continues to work.  Cannot solve the problem for several months. We use Nimble Streamer for SRT streaming to Oven, the same problem occurred when streaming from Epiphan Pearl Nano encoder

[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/11474146/ovenmediaengine.log)

**Comment by getroot** [comment]
To analyze the problem you should look at your entire ovenmediaengine.log file. In particular, logs related to OME version, stream creation, and deletion must not be deleted.

**Reply by bigsssss** [reply]
Every day a new log file is automatically created, here is the file for yesterday and today that I have. I hope this helps. 
[ovenmediaengine_20230514.log](https://github.com/AirenSoft/OvenMediaEngine/files/11478748/ovenmediaengine_20230514.log)
[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/11478751/ovenmediaengine.log)

**Reply by getroot** [reply]
Could you please provide the log generated as follows? It will be very helpful in analyzing the problem.

1. Restart OvenMediaEngine.
2. Send stream over SRT to OME.
3. Check that the problem is reproduced.

It would be great if you could upload log files from 1 to 3.

Also please share your Server.xml file and the version of OvenMediaEngine you are using so I can reproduce the problem.

**Reply by bigsssss** [reply]
I did what you said. The OME server ran for about 8 hours and the problem repeated. Video works, but no sound. I use the latest version of OME v0.15.10, but the problem was also in previous versions
[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/11483450/ovenmediaengine.log)
[ovenmediaengine20230516.log](https://github.com/AirenSoft/OvenMediaEngine/files/11483451/ovenmediaengine20230516.log)
[Server.xml.txt](https://github.com/AirenSoft/OvenMediaEngine/files/11483456/Server.xml.txt)

**Reply by getroot** [reply]
Please check if disabling WebRTC JitterBuffer in your settings solves the problem. It looks like JitterBuffer could be causing problems under certain conditions.

`<JitterBuffer>false</JitterBuffer>`

**Reply by bigsssss** [reply]
Unfortunately, this did not help and the sound stops working again after a while
[ovenmediaengine.log](https://github.com/AirenSoft/OvenMediaEngine/files/11513720/ovenmediaengine.log)
[ovenmediaengine20230519.log](https://github.com/AirenSoft/OvenMediaEngine/files/11513721/ovenmediaengine20230519.log)
[Server.xml.txt](https://github.com/AirenSoft/OvenMediaEngine/files/11513727/Server.xml.txt)

**Reply by getroot** [reply]
Thank you for supporting the test.
I registered this as an issue at https://github.com/AirenSoft/OvenMediaEngine/issues/1244. I'll try to reproduce this problem.
